symbolic-symcache/src/lib.rs

```diff
@@ -133,7 +132,8 @@ type Result<T, E = Error> = std::result::Result<T, E>;
 /// 5: PR #221: Invalid inlinee nesting leading to wrong stack traces
 /// 6: PR #319: Correct line offsets and spacer line records
 /// 7: PR #459: A new binary format fundamentally based on addr ranges
-pub const SYMCACHE_VERSION: u32 = 7;
+/// 8: PR #670: Use watto for parsing and writing
```

Comment:
Or rather: Use leb128-prefixed string table

Reply:
You're right